### DISCUSSION TITLE
Single Phase PDU with 2 Circuit Breakers

### DISCUSSION THREAD

**Discussion by jondunning** [netbox-community/netbox#7807] [Q&A] [answered]
Hi, 

What would be the best way to document a single phase PDU with power outlet and circuit breaker (bank) association?

The PDU I'm using is an APC AP8953. It's a 32a PDU but split into 2 banks, both of which have a 16a breaker. The PDU has 24 power outlets with ports 1-12 on bank 1 and ports 13-24 on bank 2. Would be good to document this if possible.

[APC AP8953 Tech Drawing here](https://download.schneider-electric.com/files?p_File_Name=VBUH-8AER8F_R0_EN.pdf&p_Doc_Ref=SPD_VBUH-8AER8F_EN&p_enDocType=Technical+Drawing)

Look forward to your suggestions/thoughts.

Thanks

**Comment by candlerb** [comment, marked as answer]
The simplest way to model this would be a PDU with 24 power outlets linked to 1 power port (inlet).  You'd lose the circuit breaker association, although if you want you could reflect this in the naming of the outlets, e.g. `OUT[1-12]-CB1` and `OUT[13-24]-CB2`

If you're trying to use Netbox data to ensure that you don't exceed the current capacity of CB1 and CB2 individually, then this won't help you.

The second way would be a PDU with 24 outlets and 2 power ports, OUT1-12 linked to IN1 and OUT13-24 to IN2. You'd then need to connect that to a fake "Y" splitter device with 2 power outlets linked to 1 power port.

**Reply by jondunning** [reply]
Makes perfect sense, thanks for spending your time explaining this!
